lib/tasks/tiny_mce_asset.rake

```diff
@@ -0,0 +1,10 @@
+namespace :tiny_mce_asset do
+  desc 'Remove obsolete images that were created on new steps or '\
+       'results and the step/result didn\'t get saved.'
+  task remote_obsolete_images: :environment do
```

Comment:
if someone generate a step and don't save it and in the process he/she creates a few images the images will get stored in our s3 but they will be obsolete see here: http://www.dictionary.com/browse/obsolete
with this task we will destroy it. I add the created_at time to be 7 days ago hopefully nobody need 7 days to edit a step description.